freq/settings.py

```diff
@@ -164,9 +164,9 @@
     'django.contrib.sites',
     'django.contrib.staticfiles',
     'lizard_auth_client',
+    'rest_framework',
```

Comment:
Even nakijken of rest_framework niet hoger in het lijstje moet (boven de standaard django dingen).